### ISSUE TITLE
As data provider I want to extract variable metadata for monolingual data sets

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#1754]

**Comment by rbirkelbach** [comment]
I was able to isolate the issue of relatedQuestionStrings getting duplicated: If one runs the routine twice within the same R session on variables with the same names, it is printed twice (or even more the more often you run the function in the same R session). 

This should not be an issue in practice, because we usually do not run into such a case and if we know that we will need to run the routine on multiple versions of the dataset (e.g. if for some reasons we decided to generate jsons for cufs, download and remote access sufs), we could simply start a new R session.

However, I am slightly confused how we should deal with this for the example data, because there we have the same datasets twice: so far I created subfolders in `data-raw/excel` and `data-raw/stata` (`./ml` `./bl` for monolingual and bilingual) and did the same for the output directory. But what about the unit-tests? Should they be run in the meantime only on the bilingual data and if we have a bugfix apply the unit tests also on the monolingual files? To my knowledge it's not possible two start separate R sessions for each testthat test, but I might be wrong? Or would the teardown already take care of this?
